src/index.ts

```diff
@@ -86,6 +81,9 @@ export const MTS_SUPPORTED_NODE_VERSION = 16
 const syncFnCache = new Map<string, AnyFn>()
 
 export interface SynckitOptions {
+  /**
+   * @deprecated bufferSize is not used anymore
+   */
   bufferSize?: number
```

Comment:
I'd remove it and release as a minor (0.x) version.

Reply:
removed